### PR TITLE
[master-2.x] Add FreeGasChangeHeight

### PR DESCRIPTION
**Pull request by Qiao-Jin**
This PR aims to add into protocol.json field `FreeGasChangeHeight` & should be used together with [neo-core-master-2.x Reset gas free #1888](https://github.com/neo-project/neo/pull/1888)

### CODE REVIEW

File: neo-cli/protocol.testnet.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "AddressVersion": 23,
     "SecondsPerBlock": 15,
     "LowPriorityThreshold": 0.001,
-    "StateRootEnableIndex": 4380100,
```
Comment (shargon) [inline neo-cli/protocol.testnet.json:7]: Why remove the previous line?

Comment (Qiao-Jin) [reply neo-cli/protocol.testnet.json:7]: typo, has fixed

**Comment by shargon** [comment]
Require update the nuget

**Comment by Qiao-Jin** [comment]
> Require update the nuget

Have updated

**Comment by superboyiii** [comment]
@erikzhang @shargon I think the time is OK.

**Review by erikzhang — APPROVED** [review]

**Pull request merged**